actioncable/lib/action_cable/subscription_adapter/test.rb

```diff
@@ -0,0 +1,24 @@
+module ActionCable
+  module SubscriptionAdapter
+    # == Test adapter for Action Cable
+    class Test < Base
```

Comment:
Can we add a `:nodoc:` and also a note saying to never use this in any environment except through tests?